cli/src/common.ts

```diff
@@ -64,14 +64,17 @@ export async function checkPackage(): Promise<string | null> {
 
 export async function checkCapacitorPlatform(
   config: Config,
-  platform: string,
+  platform: 'android' | 'ios',
```

Comment:
on every other method we use string, so for consistency I think we should keep string

maybe later on we can do a refactor to replace platform with `'android' | 'ios'`

Reply:
I agree, I was just worried somewhere it was being called for custom platforms. I'll change it back